src/main/assembly/jameica.xml

```diff
@@ -0,0 +1,23 @@
+<assembly xmlns="http://maven.apache.org/ASSEMBLY/2.2.0"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/ASSEMBLY/2.2.0 http://maven.apache.org/xsd/assembly-2.2.0.xsd">
+    <id>jameica</id>
+    <baseDirectory>${project.artifactId}</baseDirectory>
+    <dependencySets>
+        <dependencySet>
+            <outputDirectory>lib</outputDirectory>
+            <useTransitiveDependencies>false</useTransitiveDependencies>
+            <useProjectArtifact>false</useProjectArtifact>
+        </dependencySet>
+    </dependencySets>
+    <files>
+        <file>
+            <source>plugin.xml</source>
+            <filtered>true</filtered>
+        </file>
+        <file>
+            <source>${project.build.directory}/${project.build.finalName}.jar</source>
+            <destName>${project.artifactId}.jar</destName>
+        </file>
+    </files>
+</assembly>
```

Comment:
Kannst du bitte auch ein newline in der letzten Code-Zeile einfügen. Oben bei .idea/runConfigurations/JVerein.xml und glaub noch einer neuen Datei war das auch schon mal so. Wir sollten das in Zukunft mit checkstyle oder Alternativen sicherstellen.

Reply:
Man kann den Eclipse Formatter einbinden. Sowohl in IDEA als auch über Maven Plugins.

Reply:
Ist drin.